spec/lib/appsignal/integrations/mongo_ruby_driver_spec.rb

```diff
@@ -81,7 +81,7 @@
         transaction.should receive(:finish_event).with(
           'query.mongodb',
           'find | test | SUCCEEDED',
-          "{\"foo\":\"?\"}",
+          instance_of(Appsignal::Extension::Data),
```

Comment:
That's possible, but quite a bit of work. Not sure that the added value is there since we have separate tests for the sanitizer that's used.